.travis.yml

```diff
@@ -11,23 +11,18 @@ install:
 script:
   - bundle exec rake test_with_coveralls
 rvm:
-  - 1.9
```

Comment:
There were 90+ failures on 1.9, and I didn't want to focus on them.  I'll bring it back once I have it working elsewhere, and see what the problems are.  Most likely need to scope some gem versions to `platforms: [:ruby_19]` (or similar) in the Gemfile.